testing/regress/ecl/roxiepipetraced.ecl

```diff
@@ -0,0 +1,86 @@
+/*##############################################################################
```

Comment:
This needs a corresponding xml file - see the failure from the regression suite.  
Also/alternatively, because the trace context will be different each time you may need to either remove that output or mark this test as not having a key file.